templates/etc/influxdb/collectd_auth_file.j2

```diff
@@ -1,5 +1,5 @@
 # {{ ansible_managed }}
```

Comment:
Can you also change this line to:

```
{{ ansible_managed | comment }}
```

?

Reply:
So without the leading `#`

Reply:
@tersmitten yes, done